eclipse/com.gluonhq.eclipse.plugin/META-INF/MANIFEST.MF

```diff
@@ -6,7 +6,7 @@ Bundle-Version: 1.0.0.qualifier
 Bundle-Vendor: %Bundle-Vendor
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-Activator: com.gluonhq.eclipse.plugin.UiPlugin
-Require-Bundle: org.eclipse.buildship.core,
+Require-Bundle: org.eclipse.buildship.core;bundle-version="[3.0.1,4.0.0)",
```

Comment:
Does the plugin not work with Eclipse Buildship 3.0.0? That is the default version that is being installed with Eclipse 2018-12, so if there's no particular reason for using 3.0.1, we might want to lower the dependency requirement.

Reply:
I added 3.0.1 because right after they published 3.0.0 they made breaking changes.  Eclipse is shipping an artifact that was not tagged and there are significant changes a few days later in the tagged artifact. My version is actually based on the tagged repo.
I had issues installing it though, as you need to install manually 3.0.1, it doesn't ask for it.

Reply:
Yes, that is what slightly bothered me, that you need to install version 3.0.1 of Eclipse Buildship manually to be able to use this build of the Gluon plugin. As long as we clearly document it, I think this is fine.